apps/webapp/.env.example

```
@@ -26,6 +26,7 @@ SENTRY_PROJECT=
 SENTRY_AUTH_TOKEN=
 VITE_REFERRAL_CODE=
 VITE_BATCH_TX_ENABLED=
+VITE_FIXED_YIELD_MODULE_ENABLED=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_FIXED_YIELD_MODULE_ENABLED key should go before the VITE_FOOTER_LINKS key

(UnorderedKey)